### DISCUSSION TITLE
Deployment On Cloudflare Workers Dev

### DISCUSSION THREAD

**Discussion by talha-muneer** [coderaiser/cloudcmd#335] [Q&A]
Hi There,

I was wondering if it was possible to deploy cloud commander to my indexed google drive using cloudflare workers dev. Any help would be really appreciated. Thank You.

**Comment by coderaiser** [comment]
Could you please provide more details, what do you mean by `indexed google drive`?
Google drive it’s a storage, to run Cloud Commander you need `node.js` or `Docker` support.
